src/libpmemkv.cc

```diff
@@ -46,6 +46,74 @@
 #include "engines-experimental/stree.h"
 #include "engines-experimental/caching.h"
 #endif
+#include <unordered_map>
+
+typedef struct pmemkv_config_value_s pmemkv_config_value_t;
```

Comment:
we don't use typedefs almost anywhere else in PMDK, I don't see why we should begin now.
Names with `_t` are reserved in C/Posix.